Whiskey/Functions/Install-WhiskeyTool.ps1

```diff
@@ -156,7 +111,12 @@ function Install-WhiskeyTool
                                                           -Global `
                                                           -InCleanMode:$InCleanMode `
                                                           -ErrorAction Stop
-                }
+            }
+            'NuGet'
+            {   
+                $toolPath = Install-WhiskeyNuGetPackage -Name $name -Version $version -DownloadRoot $InstallRoot -ErrorAction Stop
+                $TaskParameter[$ToolInfo.PathParameterName] = $toolPath

```

Comment:
Remove this line. The PathParameterName attribute is now optional and code at the bottom of this function now handles assigning the tool path to the parameter.